docs-source/content/samples/simple/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -0,0 +1,549 @@
+{{% /notice %}}
+
+#### Create WebLogic domain
+
+  - [Create secrets](#create-secrets)
+  - [Create WebLogic Domain](#create-weblogic-domain-1)
+
+Now that we have created the AKS cluster, installed the operator, and verified that the operator is ready to go, we can have the operator create a WLS domain.
+
+##### Create secrets
+
+We will use the `kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh` script to create the domain credentials as a Kubernetes secret. Please run:
+
+```bash
+#cd kubernetes/samples/scripts/create-weblogic-domain-credentials
+$ ./create-weblogic-credentials.sh -u weblogic -p welcome1 -d domain1
+secret/domain1-weblogic-credentials created
+secret/domain1-weblogic-credentials labeled
+The secret domain1-weblogic-credentials has been successfully created in the default namespace.
+```
+
+We will use the `kubernetes/samples/scripts/create-kuberetes-secrets/create-docker-credentials-secret.sh` script to create the Docker credentials as a Kubernetes secret. Please run:
+
+```bash
+# Please change imagePullSecretNameSuffix if you change pre-defined value "regcred" before generating the configuration files.
+$ export SECRET_NAME_DOCKER="${NAME_PREFIX}regcred"
+
+#cd kubernetes/samples/scripts/create-kuberetes-secrets
+$ ./create-docker-credentials-secret.sh -s ${SECRET_NAME_DOCKER} -e oracleSsoEmail@bar.com -p oracleSsoPassword -u oracleSsoEmail@bar.com
+secret/regcred created
+The secret regcred has been successfully created in the default namespace.
+```
+
+Verify secrets with the following command:
+
+```bash
+$ kubectl get secret
+NAME                                      TYPE                                  DATA   AGE
+wlsazure-secret                           Opaque                                2      17m
+regcred                                   kubernetes.io/dockerconfigjson        1      2m25s
+default-token-csdvd                       kubernetes.io/service-account-token   3      25m
+domain1-weblogic-credentials              Opaque                                2      3m42s
+sh.helm.release.v1.weblogic-operator.v1   helm.sh/release.v1                    1      5m41s
+weblogic-operator-secrets                 Opaque                                1      5m41s
+```
+
+> **Note**: If the `NAME` column in your output is missing any of the values shown above, please reexamine your execution of the preceding steps in this sample to ensure that you correctly followed all of them.  The `default-token-mwdj8` shown above will have a different ending in your output.
+
+##### Create WebLogic Domain
+We will use the `kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain.sh` script to create the WLS domain in the persistent volume we created previously.
+
+We need to set up the domain configuration for the WebLogic domain.
+
+1. Check if resources are ready
+
+   If you used the automation script to create the AKS cluster, skip this step and go to step 2.
```

Comment:
cluster, skip -> cluster, then skip (If..., then...; please correct in the following sub-statements.)